packages/commerce-sdk-react/src/test-utils.tsx

```diff
@@ -71,13 +72,15 @@ export const renderWithProviders = (
     children: React.ReactElement,
     props?: TestProviderProps,
     options?: Omit<RenderOptions, 'wrapper'>
-): void => {
-    render(children, {
+) => {
+    const user = userEvent.setup()
+    const res = render(children, {
         wrapper: ({children}: {children?: React.ReactNode}) => (
             <TestProviders {...props}>{children}</TestProviders>
         ),
         ...options
     })
+    return {user, ...res}
```

Comment:
Why did you choose to spread the res object?

Reply:
To avoid breaking the tests that use return of render function. Is there a good reason to not spread this? On the document site, it seems to recommend so
https://testing-library.com/docs/user-event/intro/   

```
const {container} = render(<MyComponent />

```